src/raven.js

```diff
@@ -759,7 +760,14 @@ function makeRequest(data) {
     var img = newImage(),
         src = globalServer + authQueryString + '&sentry_data=' + encodeURIComponent(JSON.stringify(data));
 
-    img.crossOrigin = 'anonymous';
+    if ('crossOrigin' in globalOptions) {
```

Comment:
I'd say either remove this check, or remove the default. I think it makes more sense here to remove the default.

I'd rather let this feature go undocumented for the time being, since it'd make more sense to allow this to be done through a new transport, but that hasn't been fully thought out yet for raven.js.

Sound reasonable?